spec/controllers/spree/users_controller_spec.rb

```diff
@@ -27,6 +38,14 @@
         expect(assigns[:user].email).to eq 'mynew@email-address.com'
         expect(response).to redirect_to spree.account_path
       end
+
+      context 'and locale is not default' do
+        it 'performs update' do
```

Comment:
Split these two expect statements into their own tests. you can have the "put :update ..." as a before clause that gets run before each test. Generally, testing best practices say that [each test should assert only one thing](https://www.betterspecs.org/#single)